src/components/TextField/TextField.tsx

```diff
@@ -7,6 +7,7 @@ type Props = {
   label?: string,
   required?: boolean,
   onChange?: (newValue: string) => void,
+  isValid?: (url: string) => boolean,
```

Comment:
```suggestion
  validate?: (value: string) => boolean,
```

Use proper naming for function. Also, it can be any value, not only URL.